root/account/registered_discourse.tt

```diff
@@ -0,0 +1,10 @@
+[% WRAPPER 'layout.tt' title=l("Account Created") %]
```

Comment:
This page will be displayed if the user just created an account and the return uri matches `^/discourse/sso`.